columnize.go

```diff
@@ -65,18 +67,22 @@ func getWidthsFromLines(config *Config, lines []string) []int {
 // returns a sprintf-style format string which can be used to print output
 // aligned properly with other lines using the same widths set.
 func (c *Config) getStringFormat(widths []int, columns int) string {
+	// Create the buffer
+	// 8 bytes per character * 8 characters per column * ~num columns
+	buf := bytes.NewBuffer(make([]byte, 0, 8*len(widths)*8))
```

Comment:
This should never really be this large because it is just the formatting of a single line, sans data. I would guess the size would end up being closer to `(6 + len(c.Glue)) * columns` (`%-NNNs<glue>` per-column). I'm fine with it though if there is some subtle reason the math is faster this way.